src/raglite/_extract.py

```diff
@@ -52,6 +52,7 @@ class MyNameResponse(BaseModel):
                 "name": return_type.__name__,
                 "description": return_type.__doc__ or "",
                 "schema": return_type.model_json_schema(),
+                "strict": True,
```

Comment:
For my information, why is strict mode desirable here? (I think I know the answer but would like some validation that it's what I think it is :-)).

Reply:
Without strict mode the generation is wrong, it mixes the schema and the generation and then Pydantic fails.